src/placeos-models/control_system.cr

```diff
@@ -84,12 +84,8 @@ module PlaceOS::Model
 
     # Validate support URI
     validate ->(this : ControlSystem) {
-      support_url = this.support_url
-      unless support_url.empty?
-        url = URI.parse(support_url)
-        url_parsed = !!(url && url.scheme && url.host)
-        this.validation_error(:support_url, "is an invalid URI") unless url_parsed
-      end
+      return if this.support_url.blank?
```

Comment:
There is already blank check in Validation.valid_uri?

Reply:
it's valid to have a blank support URL